sklearn/gaussian_process/gpr.py

```diff
@@ -140,6 +141,12 @@ def __init__(self, kernel=None, alpha=1e-10,
         self.copy_X_train = copy_X_train
         self.random_state = random_state
 
+    @property
+    @deprecated("Attribute ``rng_`` is deprecated in version 0.19 and will be"
+                " used only as a variable in version 0.21.")
```

Comment:
Added it from a developer point of view. But yes it would be unclear. Making the change.